src/types.rs

```diff
@@ -296,8 +337,9 @@ impl Signature {
     /// ```
     /// # use term_rewriting::{Signature};
     /// let mut sig = Signature::default();
-    /// let z = sig.new_var(Some("Z".to_string()));
-    /// let z2 = sig.new_var(Some("Z".to_string()));
+    ///
+    /// let z = sig.new_var(Some("z".to_string()));
+    /// let z2 = sig.new_var(Some("z".to_string()));
     ///
     /// assert_ne!(z,z2);
```

Comment:
It looks like there's a spacing issue here and in some other tests (i.e. `(z,z2)` instead of `(z, z2)`). Can you fix those please?